apps/chat/src/components/Chatbar/ModelIcon.tsx

```diff
@@ -14,10 +14,11 @@ import Tooltip from '@/src/components/Common/Tooltip';
 interface Props {
   entityId: string;
   entity: DialAIEntity | undefined;
-  size: number;
+  size: number | string;
```

Comment:
I don't think it's good idea to allow strings here

Reply:
![Screenshot 2024-09-12 at 10 55 18](https://github.com/user-attachments/assets/d2b750f6-aaa3-4b9e-a1f5-636d4d584a29)
It the easiest way to use `%`. library allows it